contracts/core/SetToken.sol

```diff
@@ -184,17 +177,7 @@ contract SetToken is
         // Check that function caller is Core
         require(msg.sender == ISetFactory(factory).core(), "ONLY_CORE_CAN_BURN_SET");
 
-        // Require user has tokens to burn
```

Comment:
We still want this check right?

Reply:
It's actually checked already in the `_burn` function of ERC20.sol

Reply:
Oh gotcha, see my other comment